file-proxy/README.md

```diff
@@ -0,0 +1,175 @@
+
+## Included examples
+[simple-file-proxy](#simple-file-proxy)  
+[multiple-file-proxy](#multiple-file-proxy)  
+[metadata-file-proxy](#metadata-file-proxy)  
+[extension-file-proxy](#extension-file-proxy)  
+[encrypted-file-proxy](#encrypted-file-proxy)  
+[auth-file-proxy](#auth-file-proxy)
+
+## Getting started
+
+### Setup Graphcool project
+
+Make sure you have the Graphcool cli installed:
+```sh
+npm -g install graphcool
+```
+
+### Setup webtask.io
+Install the webtask.io cli, and initialize your webtask.io profile, by running:
+```sh
+npm -g install wt-cli
+wt init
+```
+If you have never used wt-cli before, the `wt init` command will ask for your e-mail address or phone number to set up or connect to your webtask container.
+
+## Examples
+### simple-file-proxy
+
+This is the starting point for the examples. It is a basic file upload/download proxy.
+
+First, initialize a new Graphcool project with the required schema, by running:
+```sh
+graphcool init --schema ./schemas/simple-file-proxy.graphql
+```
+Note the ID that this command returns. We will need this project ID later on.
+
+You can deploy the webtask for this example to webtask.io by running the following command:
+```sh
+wt create ./simple-file-proxy.js --name simple
+```
+After running this command, `wt-cli` will return the url of your webtask. Using the `-name` argument, you can specify a different name for the webtask (the last part of the url).
+
+Your webtask is now deployed! You can upload a file using any tool that support `multipart/form-data`, like Postman, or the command-line tool `curl`:
+```sh
+curl -X POST '<webtask endpoint url>/__PROJECT_ID__' -F "data=@example.png;filename=myname.png"
+```
+This uploads the local file `example.png` with the new name `myname.png`. The response should look something like this:
+```json
+{
+  "name": "myname.png",
+  "size": 123456,
+  "newUrl": "https://<webtask endpoint url>/__PROJECT_ID__/__SECRET__",
+  "id": "<omitted>",
+  "contentType": "image/png"
+}
+```
+Notice that the `url` field in the response is replaced by a new field, `newUrl`, that points to your own endpoint for downloading the file.
+You can open the url from the `newUrl` field in your browser to download the file you have just uploaded.
+
+### multiple-file-proxy
+
+This example adds the possibility to upload multiple files in one request.
+
+This example uses the same GraphQL schema as `simple-file-proxy`. Follow the instructions above if you haven't created a Graphcool project based on this schema.
+
+Next, deploy the webtask for this example to webtask.io by running the following command:
+```sh
+wt create ./multiple-file-proxy.js --name multiple
+```
+Now, try to upload more than one file at once to your endpoint:
+```sh
+curl -X POST '<webtask endpoint url>/__PROJECT_ID__' -F "file1=@example.png;filename=myname.png;file2=@example2.png;filename=testfile.png"
```

Comment:
The syntax of the curl command is wrong. It requires to seperate -F sections. I will update the readme.